pootle/apps/pootle_fs/forms.py

```diff
@@ -18,6 +18,17 @@
     fs_plugins, fs_translation_mapping_validator, fs_url_validator)
 
 
+FS_CHOICES = (
+    ("gnu", _("GNU-style"), "/po/<language_code>.<ext>"),
```

Comment:
i think as its an example - lets use `po` - then users can use custom if they need to change